src/internal/Component.js

```diff
@@ -48,7 +48,7 @@ function make_dirty(component, key) {
 	if (!component.$$.dirty) {
 		dirty_components.push(component);
 		schedule_update();
-		component.$$.dirty = {};
+		component.$$.dirty = Object.create(null);
```

Comment:
Not sure if this should be using the `blank_object` helper function, or maybe just get rid of that function, since it's not being used much anyways.

Reply:
Hah, I didn't see that helper. Really strange that it's not used for this object.

Reply:
It's also only used in two places, so I'm not sure if we should just get rid of it.

https://github.com/sveltejs/svelte/blob/d03eddcce266ba519d3828226b2d38e7a5b9fe54/src/internal/Component.js#L70
https://github.com/sveltejs/svelte/blob/2fc07c6dfb652076d3d91287ae62018fdd3c0715/src/internal/ssr.js#L82